src/index.js

```diff
@@ -117,13 +117,15 @@ module.exports = exports = options => {
   async function broadcast(data, options, clients) {
     options = options || {}
     const rms = options.rooms || []
-    if (rms.length === 0) {
+    let ids = []
```

Comment:
I like the idea and we can return the sent set as well. I can implement this approach tomorrow when I get the chance